src/robot/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+#==============================================================================
+# Dependencies
+#
+
+include(CheckCXXSourceCompiles)
```

Comment:
Are these necessary if we're not specifying any dependencies? It seems that most of the other `CMakeLists.txt` just start with defining `sources`. @jslee02